### ISSUE TITLE
Connection fails for user with no password

### ISSUE THREAD

**Issue by kodeo** [bgrainger/MySqlConnector#53]
var conn = new MySqlConnection("Server=localhost;Database=mydb;Uid=user1;");
    conn.Open();

gets:

```
An unhandled exception of type 'MySql.Data.MySqlClient.MySqlException' occurred in System.Private.CoreLib.ni.dll
 
Additional information: Access denied for user 'user1'@'localhost' (using password: YES)
```

**Comment by bgrainger** [comment]
Fixed in [0.1.0-alpha20](https://www.nuget.org/packages/MySqlConnector/0.1.0-alpha20).

**Comment by kodeo** [comment]
Wow, quick work. Thanks.

**Comment by bgrainger** [comment]
Needs to be tested with MariaDB 10.1.17 with Windows 10 Anniversary Update.

**Comment by bgrainger** [comment]
Fixed in [0.1.0-alpha22](https://www.nuget.org/packages/MySqlConnector/0.1.0-alpha22) for MariaDB.
